utils/issue-format-bot/test/mocks/context.js

```diff
@@ -22,6 +22,12 @@ module.exports = {
       github: {
         issues: {
           createComment: sinon.spy(),
+          // TODO actually make this keep track of data
+          getComments: function() {
+            return {
+              data: []
+            };
+          },
           addLabels: sinon.spy(),
           removeLabel: sinon.spy(() => new Promise(noop))
```

Comment:
`removeLabel: sinon.spy(() => Promise.resolve())`?

Reply:
I think `new Promise(noop)` makes it clearer what's going on - we're creating a new Promise that's a noop. Whereas with your example you have to know offhand what `Promise.resolve()` does.

Reply:
`new Promise(noop)` never resolves since `resolve` is never called.